src/cli/options.ts

```diff
@@ -186,7 +192,11 @@ export const uploadFolderOptions = [
   optionMap.maxConcurrency,
 ];
 
-export const uploadFileOptions = [...uploadOptions, optionMap.filePath];
+export const uploadFileOptions = [
+  ...uploadOptions,
+  optionMap.filePath,
+  optionMap.tags,
+];
```

Comment:
for above, can add optionMap.tags instead to uploadOptions to include option on upload-folder